src/table/table.scss

```diff
@@ -4,383 +4,373 @@
-      @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
-      }
+    > .iui-filter-button {
+      margin-left: $iui-xs;
 
-      > .iui-tables-slot > .iui-tables-menu {
-        visibility: visible;
+      &:not(.iui-active) {
+        visibility: hidden;
       }
     }
 
-    > .iui-tables-cell {
-      align-items: center;
-      display: flex;
-      flex-grow: 1;
-      min-width: 145px;
-      padding: $iui-table-density;
-      overflow: hidden;
-      list-style: none;
-      flex-basis: 145px;
-
-      > svg {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin-right: $iui-sm;
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
       }
 
-      > .iui-status,
-      > .iui-progress-indicator-radial {
-        float: right;
-        margin-left: auto;
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
       }
+    }
 
-      > .iui-status {
-        margin-right: $iui-xs;
-      }
+    &:hover > .iui-resizer {
+      display: block;
+    }
 
-      > .iui-user-icon {
-        margin-right: $iui-s;
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
       }
 
-      &.iui-tables-slot {
-        width: $iui-l * 2;
-        padding: 0;
-        flex-grow: 0;
-        min-width: 0;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex-basis: $iui-l * 2;
-
-        > .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-            color: t(iui-icons-color-actionable);
-          }
-          width: $iui-icons-default;
-          height: $iui-icons-default;
-          margin: 0;
-        }
+      .iui-sort,
+      .iui-filter-button {
+        visibility: visible;
       }
+    }
+  }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
-
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
-
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+  // Sort icon
+  .iui-sort {
```

Comment:
When we added sort to react, we talked that we need to make not sorted icon different by changeing opacity or different icon and changing its fill.